cadl-extension/src/main/java/com/azure/autorest/CadlPlugin.java

```diff
@@ -189,4 +178,8 @@ public void message(Message message) {
                 break;
         }
     }
+
+    private String getOutputFolder() {
+        return (String) SETTINGS_MAP.get("output-folder");
```

Comment:
nit, seems easier to put the EmitterOptions in class var?

Reply:
sure